eq-author/src/components/CommentsPanel/commentsQuery.graphql

```diff
@@ -0,0 +1,29 @@
+query Page($input: QueryInput!) {
+  page(input: $input) {
+    id
+    comments {
+      id
+      commentText
+      user {
+        id
+        name
+        picture
+        email
+        displayName
+      }
+      createdTime
+      replies {
+        id
+        commentText
+        createdTime
+        user {
+          id
+          name
+          picture
+          email
+          displayName
+        }
+      }
+    }
+  }
+}
```

Comment:
I think there should be a newline at the end of this file.

Reply:
now removed as not implementing delete comments yet

Reply:
Both have lines! - good to know